src/content/docs/reference/api/git-credentials.md

```
@@ -0,0 +1,69 @@
+protocol=https
+host=github.com
+path=owner/repository
+```
+
+## Response format
+
+### Success response (200 OK)
+
+When a token is successfully vended, the response contains Git credential helper output:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)